Automattic/woocommerce-payments
# E2E Test for Partial Refunds

Closes https://github.com/Automattic/woocommerce-payments/issues/2212

#### Changes proposed in this Pull Request

Title: E2E Test for the `Merchant / Order / Partial refunds (one, several)` critical flow

Description: This PR contains two test cases for partial refunds:
1. A test case where one of two line items is partially refunded by refunding a portion of its cost, like shown below:
    <img width="678" alt="Screen Shot 2021-07-22 at 11 41 57 PM" src="https://user-images.githubusercontent.com/4509348/126668483-120e88a4-8b91-4ff5-81f8-8b8d15c85a93.png">
2. A test case where two of three line items are refunded. The first line item is fully refunded, while in the second line item, only one piece was refunded. Something like below:
    <img width="997" alt="Screen Shot 2021-07-22 at 11 43 45 PM" src="https://user-images.githubusercontent.com/4509348/126668674-5180852e-ffe8-40fa-b61b-568ff1ed1925.png">

#### Testing instructions
1. Setup the e2e testing environment as usual.
2. Run the newly added test: `npm run test:e2e ./tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js`. These two tests should pass:
    <img width="510" alt="Screen Shot 2021-07-23 at 12 10 23 AM" src="https://user-images.githubusercontent.com/4509348/126672234-2953bd89-604a-4c70-bd30-76a7e466d438.png">
